dom-bom-weather/index.html

```diff
@@ -8,14 +9,20 @@
     <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
     <title>Search the weather</title>
 </head>
+
 <body>
     <main>
-        <label class="text" for="weatherSearch">Type a city</label>
-        <input type="search" id="weather_Search" name="weathe" placeholder="Enter the city" class="input" />
-        <button id='button'>Find</button>
-        <div id="search"></div>
-        <div id ='output'></div>
+        <div class="input_block">
+            <label class="text" for="weatherSearch">Type a city</label>
+            <input type="search" id="weather_Search" name="weathe" placeholder="Enter the city" class="input" />
+            <button id='button'>Find</button>
```

Comment:
Id's should describe the element widely, for example:
```suggestion
            <button id='search_button'>Find</button>
```
but if you will use form `submit` event you won't need this id at all